README example no longer compiles

```
error[E0107]: struct takes 1 generic argument but 2 generic arguments were supplied
   --> src/main.rs:5:16
    |
5   |     let mut v: SmallVec<i32, 4> = smallvec![1, 2, 3, 4];
    |                ^^^^^^^^      - help: remove this generic argument
    |                |
    |                expected 1 generic argument
    |
note: struct defined here, with 1 generic parameter: `A`
   --> /playground/.cargo/registry/src/index.crates.io-6f17d22bba15001f/smallvec-1.13.1/src/lib.rs:768:12
    |
768 | pub struct SmallVec<A: Array> {
    |            ^^^^^^^^ -
```

--- a/README.md
+++ b/README.md
@@ -28,7 +28,7 @@ rust-smallvec
 use smallvec::{SmallVec, smallvec};
     
 // This SmallVec can hold up to 4 items on the stack:
-let mut v: SmallVec<i32, 4> = smallvec![1, 2, 3, 4];
+let mut v: SmallVec<[i32; 4]> = smallvec![1, 2, 3, 4];
 
 // It will automatically move its contents to the heap if
 // contains more than four items:

